packages/components/src/components/marketplace-search.tsx

```diff
@@ -35,18 +36,17 @@ export const MarketplaceSearch = ({
   const items = useMemo(() => {
     let results = null;
     if (query && queryList) {
-      const tagsFilter = query.split(' ').filter(t => t.trim().length > 1 && t.startsWith('#'));
```

Comment:
The tag filter search feature no longer works on the envelop plugins page.

See
https://the-guild.dev/graphql/envelop/plugins



https://user-images.githubusercontent.com/4893048/210891655-f0ab2f64-b5f4-4911-b3d9-020c8b8005ac.mp4